### ISSUE TITLE
Pretrained model for novel category discovery on ImageNet

### ISSUE THREAD

**Issue by QingC0** [k-han/AutoNovel#3]
Hi,
Your work impressed me! I would like to see how your model evaluated on the third step, new category discovery on ImagNet. 
I tried to download your pretrained model and train on imagenet as your step 3 described. But can't get a good result. Can you upload your pretrained model for the training part in the step 3 for imagenet?

**Comment by k-han** [comment]
Hi,  the trained models and all other required pretrained models have already been released. You will find them after running: 
> sh scripts/download_pretrained_models.sh 

The pretrained models for ImageNet experiments are placed in: 
> pretrained/auto_novel_imagenet

Note that there are three models corresponding to three different splits as illustrated in the paper.

**Comment by QingC0** [comment]
Thanks for your reply!
In your paper, when you say how you pretrain the model for Imagenet, "we follow (Han et al., 2019) and use metric learning on the labelled classes to pre-train the feature extractor, instead of the self-supervised learning". I am curious how did the metric learning exactly conduct? ( I read the code released by (Han et al., 2019), but only to find it's just a supervised learning for pretraining.

**Comment by k-han** [comment]
For the ImageNet pre-trained model on 882 classes of labelled data, it is simply obtained by training with cross-entropy loss. It is just a plain classification model. However, the pretraining can be done by any form of metric learning including cross-entropy loss, triplet loss or any other suitable losses. Practitioners can take whichever they like.

In DTC paper, for fair-comparison with KCL and MCL, I obtained the pre-trained 882-way classification model from KCL&MCL authors and converted them into the Pytorch (`resnet18_imagenet_classif_882_ICLR18.pth`). You can find this model after downloading following the instructions. 

Meanwhile, in DTC, I have another experiment dealing with an unknown number of classes, where I trained a 800-way classifier from scratch, which gave reasonably good initial representation: https://github.com/k-han/DTC/blob/master/imagenet_classif.py Again, there is nothing special. It is just a plain classification model trained with cross-entropy loss.

**Comment by QingC0** [comment]
Thanks a lot!

**Comment by QingC0** [comment]
So when using ImageNet data, the 3 steps in your paper (self-supervised learning, supervised learning with labelled data, and joint optimization) become 2 steps without the first step since that is not helpful?

**Comment by k-han** [comment]
yep

**Comment by QingC0** [comment]
Thank you very much
